### PR TITLE
WRP-14791: Fixed LabeledIconButtons css styling on console app

### PR DESCRIPTION
**Pull request by ion-andrusciac-lgp**
### Checklist 

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed LabeledIconButton css styling on console app

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
On the `Silicon` skin LabeledIconButtons from the `Drawer` was resized to make them look like in the other skins

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Changed the `Cell` height for the `Cobalt` and `Copper` skin to make the entire left menu visible

### Links
[//]: # (Related issues, references)
WRP-14791

### Comments

Enact-DCO-1.0-Signed-off-by: Ion Andrusciac ion.andrusciac@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

File: console/src/components/Widget/Widget.module.less [hunk 1]
```diff
@@ -1,8 +1,38 @@
 @import "@enact/agate/styles/variables.less";
+@import "@enact/agate/styles/skin.less";
```
Comment (stanca-pop-lgp) [inline console/src/components/Widget/Widget.module.less:1]: Please sort imports.

File: console/src/components/Widget/Widget.module.less [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+.applySkins({
+    :global(.silicon) {
+        .iconLabel {
+            padding-bottom: (@agate-labelediconbutton-small-icon-button-height / 2);
+        }
+
+        .labeledIconButton {
+            &.large {
+                padding: 0;
+                &.below {
```
Comment (stanca-pop-lgp) [inline console/src/components/Widget/Widget.module.less:21]: Please leave an empty row above.

File: console/src/App/App.module.less [hunk 3]
```diff
@@ -44,6 +44,13 @@
 :global(.cobalt),
 :global(.copper) {
 	.app {
+		// This method was used to make all items in the left menu to be visible
```
Comment (stanca-pop-lgp) [inline console/src/App/App.module.less]:
```suggestion
		// This method was used to make all items in the left menu visible
```

**Review by stanca-pop-lgp — COMMENTED** [review]
Please check my comments.

**Review by stanca-pop-lgp — APPROVED** [review]

**Pull request merged**